### PR TITLE
chore: ignore generated files in pre-commit hook

### PR DESCRIPTION
**Pull request by u2**

### CODE REVIEW

File: devtools/git/pre-commit [hunk 1]
```diff
@@ -85,7 +100,7 @@ else
   FMTFAILED=0
   for file in $(git diff --name-only --cached);
   do
-    if [ -f "$file" ] && [ ${file: -3} == ".rs" ]; then
+    if [ -f "$file" ] && [ ${file: -3} == ".rs" ] && [ $(contains "${IGNORE[@]}" "$file") == "n" ] ; then
```
Comment (doitian) [inline devtools/git/pre-commit]: I recommend using `case` here to filter out files that should be ignored.

Comment (u2) [reply devtools/git/pre-commit]: updated

**Comment by doitian** [comment]
ping @nervosnetwork/ckb-dev

**Pull request merged**